buildscripts/kokoro/cronet.sh

```diff
@@ -3,6 +3,119 @@
 set -exu -o pipefail
 cat /VERSION
 
-cd ./github/grpc-java/cronet
+BASE_DIR=$(pwd)
+
+# Set up APK size and dex count statuses
+
+gsutil cp gs://grpc-testing-secrets/github_credentials/oauth_token.txt ~/
```

Comment:
It seems the code needs to handle KOKORO_GITHUB_PULL_REQUEST_COMMIT being unset. This script can be run on master/release branches.